### ISSUE TITLE
New content dosen’t get synced to Kodi.

### ISSUE THREAD

**Issue by coz2001** [croneter/PlexKodiConnect#748]

**Comment by eifelt** [comment]
For me the same. Revert to Version before and alle runs fine.

**Comment by croneter** [comment]
Does new content get synched if you restart Kodi? (=do a full sync)

**Comment by eifelt** [comment]
no. i had reset the db, does a manual sync, but no updates. In the plex addon under recent films the movies appear. but not in the overview .. i had 2 films h264 that wont play with the new Version.

**Comment by coz2001** [comment]
PKC 2.7.3
Restart Kodi don't help.
Perform manual sync don't sync new content.
Reset DB don't sync the new content only items that already was in the Kodi DB before the reset.
The new content don't show anywhere neither in the Plex add-on.
Choose Plex server from a list show my server as a (remote) which is wrong.

I'm not sure if the issue got with 2.7.2 or 2.7.3.
I tried to go back to 2.7.1 and reset the DB and that did pick up the new items.

It do annoy me that my server shows as remote I never seen that before.

Let me know if you need anything else.

**Comment by croneter** [comment]
Hmm, weird. I don't see anything in the log, unfortunately. Do you happen to have an old log that you could also upload? (does not need to be debug, necessarily, if PKC crashed) 

Could you please capture a debug log when you reset the Kodi database?

**Comment by coz2001** [comment]
Sadly I don't have any old logs.

[Resetting.log](https://github.com/croneter/PlexKodiConnect/files/2906154/Resetting.log)

[Rebuilding.log](https://github.com/croneter/PlexKodiConnect/files/2906156/Rebuilding.log)

**Comment by croneter** [comment]
Thanks for the new logs, @coz2001 . The "resetting" one is not really relevant as it won't fix your problem for sure. The "rebuilding" one is, though, because it rebuilds your entire library from scratch.

However, there is nothing in the log indicating that something went wrong.
* Are you sure that items are now missing on the Kodi side of things? If so, which items are missing?
* Are always the same items missing when you reset the Kodi database?

**Comment by coz2001** [comment]
This is so strange, I can see the items in the rebuild log but they still don't show up in Kodi (estuary).

I will try rebuild again with 2.7.1

**Comment by croneter** [comment]
You have many different Plex libraries. Do the items show up under "Videos" - so outside of your widgets that you probably have tuned to the different libraries?

**Comment by coz2001** [comment]
No they don't show under videos.

**Comment by croneter** [comment]
Mind sharing your library with me (croneter)? I'll see whether I can reproduce your issue - much easier fixing then 😉

**Comment by coz2001** [comment]
@croneter when did you release 2.7.3

**Comment by croneter** [comment]
"2 days ago": https://github.com/croneter/PlexKodiConnect/releases

**Comment by coz2001** [comment]
Okay, rebuild on 2.7.1 and they all show in recently added.

The last item that showed on 2.7.3 was 2 days ago which was when you released it, so there has to be a bug with 2.7.3 some where.

I will add you my friend.

**Comment by croneter** [comment]
Hold your horses - do you have several "MyVideosXXX.db" in your `Database` directory in the [Kodi userdata folder](https://kodi.wiki/view/Userdata)? 

PKC changed the way they are picked up: 
* Kodi 18: PKC will pick `MyVideos116.db` (=stable Kodi 18.0, 18.1). If that file is not around, it'll pick `MyVideos113.db` (=an earlier Kodi 18 Release Candidate)
* Kodi 17: PKC will only pick `MyVideos107.db`

**Comment by coz2001** [comment]
Yes

<img width="186" alt="screenshot 2019-02-26 at 17 22 38" src="https://user-images.githubusercontent.com/15858702/53428482-2ad1c880-39eb-11e9-86b0-235edad6b31c.png">

**Comment by croneter** [comment]
😆 
Well that was silly of me:
```
NOTICE: PLEX.service: Kodi video database version: 113
NOTICE: PLEX.service: Kodi music database version: 72
NOTICE: PLEX.service: Kodi texture database version: 13
```
Forgot to include a `break` there in the code 🙈 

# Workaround for now
* Make sure Kodi does not run. 
* Do make a backup, just in case. Then just delete the `MyVideos113.db` (and maybe the other `MyVideos113.*` in the folder `Database` in the [Kodi userdata folder](https://kodi.wiki/view/Userdata)

That's it...

**Comment by coz2001** [comment]
👍 Happy to hear you found the bugger...

**Comment by coz2001** [comment]
You solution worked, rebuild on 2.7.3 and recently added is there, thanks.....

**Comment by herrwusel** [comment]
Can confirm that this fix is working. It also fixes not playing some content (guess it has to do with not updating properly).

**Comment by croneter** [comment]
Fixed with stable and beta 2.7.4
